lib/helpers/stake/gateway_composer/StakeHelper.js

```diff
@@ -379,7 +394,6 @@ class StakeHelper {
     const defaultOptions = {
       from: owner,
       to: oThis.gatewayComposer,
-      gas: '7500000',
```

Comment:
Why tx-options are required for raw tx? 🤔

Reply:
They are being passed into the constructor of `Contract`, so they are still being used. Changing it more is out of the scope of this PR in my opinion (considering that this class is also going to be deprecated).